docs/kusama-parameters.md

```diff
@@ -1,57 +1,74 @@
+See [Polkadot Parameters](kusama-time-periods) for more details on how Kusama's parameters differ
+from Polkadot's._
 
-- Block: 6 seconds
+- Slot / Block: 6 seconds (although see note below)
 - Epoch: 1 hour (600 slots x 6 seconds)
 - Session: 1 hour (6 sessions per Era)
 - Era: 6 hours (3600 slots x 6 seconds)
 
 | Kusama  | Time      | Slots\* |
 | ------- | --------- | ------- |
-| Block   | 6 seconds | 1       |
+| Slot    | 6 seconds | 1       |
 | Epoch   | 1 hour    | 600     |
 | Session | 1 hour    | 600     |
 | Era     | 6 hours   | 3,600   |
 
-\*_A maximum of one block per slot can be in a chain._
+\*_A maximum of one block per slot can be in a canonical chain. However, occasionally a slot will be
+without a block in the chain. Thus, the times given are estimates. See [Consensus](learn-consensus)
+for more details._
 
 ### Governance
 
 | Democracy        | Time   | Slots   | Description                                                                                                                                                  |
 | ---------------- | ------ | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| Voting period    | 7 days | 604,800 | How long the public can vote on a referendum.                                                                                                                |
-| Launch period    | 7 days | 604,800 | How long the public can select which proposal to hold a referendum on. i.e., Every week, the highest-weighted proposal will be selected to have a referendum |
-| Enactment period | 8 days | 691,200 | Time it takes for a successful referendum to be implemented on the network.                                                                                  |
+| Voting period    | 7 days | 100,800 | How long the public can vote on a referendum.                                                                                                                |
+| Launch period    | 7 days | 100,800 | How long the public can select which proposal to hold a referendum on. i.e., Every week, the highest-weighted proposal will be selected to have a referendum |
+| Enactment period | 8 days | 115,200 | Time it takes for a successful referendum to be implemented on the network.                                                                                  |
 
 | Council       | Time  | Slots  | Description                                                          |
 | ------------- | ----- | ------ | -------------------------------------------------------------------- |
 | Term duration | 1 day | 14,400 | The length of a council member's term until the next election round. |
 | Voting period | 1 day | 14,400 | The council's voting period for motions.                             |
 
+The Kusama Council consists of up to 19 members and up to 19 runners-up.
```

Comment:
```suggestion
The Kusama Council consists of up to 19 members and up to 19 runners up.
```
according to [this page on council](https://polkadot.js.org/apps/#/council), it's just spelt "runners up"